Recognize NSS bridge interfaces on IPQ5018 independently of the PPE backend.

--- a/nss_qdisc/Makefile
+++ b/nss_qdisc/Makefile
@@ -2,8 +2,13 @@ ccflags-y := -I$(obj)/../exports -I$(src
 
 # If NSS_QDISC_PPE_SUPPORT is removed, then remove/disable
 # nss_ppe.o nss_ppe_mc.o from qca-nss-qdisc-objs list.
+ifneq ($(NSS_QDISC_PPE_ENABLE),n)
 ifeq ($(SoC),$(filter $(SoC),ipq807x ipq807x_64 ipq60xx ipq60xx_64))
-ccflags-y += -DNSS_QDISC_PPE_SUPPORT -DNSS_QDISC_BRIDGE_SUPPORT
+ccflags-y += -DNSS_QDISC_PPE_SUPPORT
+endif
+endif
+ifeq ($(NSS_QDISC_BRIDGE_ENABLE),y)
+ccflags-y += -DNSS_QDISC_BRIDGE_SUPPORT
 endif
 
 ccflags-y += -Wall -Werror
@@ -22,9 +28,11 @@ qca-nss-qdisc-objs := 	nss_bf.o \
 			nss_wred.o \
 			nss_wrr.o
 
+ifneq ($(NSS_QDISC_PPE_ENABLE),n)
 ifeq ($(SoC),$(filter $(SoC),ipq807x ipq807x_64 ipq60xx ipq60xx_64))
 qca-nss-qdisc-objs += 	nss_ppe.o \
 			nss_ppe_mc.o
 endif
+endif
 
 obj-$(igs)+= igs/
